Ports/poppler-data/package.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='poppler-data'
```

Comment:
How are `poppler` and `poppler-data` related to each other? Can't `poppler` just simply download and install the data as well? And if not, shouldn't `poppler` have a dependency on `poppler-data`?